manager/manager.go

```diff
@@ -919,8 +919,9 @@ func (m *Manager) becomeLeader(ctx context.Context) {
 	initialCAConfig.ExternalCAs = m.config.ExternalCAs
 
 	var (
-		unlockKeys []*api.EncryptionKey
-		err        error
+		unlockKeys   []*api.EncryptionKey
+		err          error
+		vxlanUDPPort uint32 = 4789
```

Comment:
Same as above, couldn't we try to define this in only one place (preferably in libnetwork)?